### ISSUE TITLE
Missing nucleus in nuclei segmentation

### ISSUE THREAD

**Issue by AnnaJouannic** [MouseLand/cellpose#269]
Hello,

I am trying to run nuclei segmentation on my jupyter notebook. I am using the following code on a .tif image : 

model = models.Cellpose(gpu = False, model_type = 'nuclei', torch = True)
channels = [0,0]
masks, flows, styles, diams = model.eval(image, diameter=20, channels=channels)

I get the result below. Some nucleus are not recognized. I tried to change the diameter or turn off flow_threshold but it does not improve the segmentation.
Have you got any idea of why some nucleus are missing ? 

Thanks for this tool and for your help.

![t_cp_output](https://user-images.githubusercontent.com/79096671/119521730-b463d280-bd7b-11eb-9f6b-4cb00539f1b6.png)

**Comment by asanborn** [comment]
We are seeing the same problem of missing nuclei:

![image](https://user-images.githubusercontent.com/10459870/120030899-e1e39280-bfac-11eb-88f4-88dffca50e27.png)

It seems that the brightest nuclei are consistently the ones that are dropping out, and are even missing entirely in the cell pose flow predictions (right-most image). We managed to get around this by doing a gamma correction, which brings the bright and dim nuclei closer in intensity values. Background subtraction (e.g. 25th percentile intensity) can also help a bit. Gamma = 1/3 worked well for us, and this is the result:

![image](https://user-images.githubusercontent.com/10459870/120031384-95e51d80-bfad-11eb-9d0b-315c8498ec0d.png)

**Comment by AnnaJouannic** [comment]
Thank you for the answer.
I apply a gamma correction (0.3) and it work well, nuclei that were not recognized are now recognized.
Here is the result : 
![cyto_cp_output](https://user-images.githubusercontent.com/79096671/121522919-e9b42580-c9f5-11eb-90a7-cdb18cec3362.png)


I try to run the segmentation with 'cyto' method, without the gamma correction and the result is quite similar as above : 
![cyto_cp_output](https://user-images.githubusercontent.com/79096671/121523303-5a5b4200-c9f6-11eb-9e79-59aaff8e28cf.png)

**Comment by carsen-stringer** [comment]
glad you found some preprocessing that worked well for you!
